bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -401,7 +420,9 @@ public void run() {
         }
     }
 
-    public void runWithFlags(boolean force, boolean suspendMajor, boolean suspendMinor) {
+    public void runWithFlags(boolean force, boolean suspendMajor, boolean suspendMinor,
+                             double majorCompactionThreshold, double minorCompactionThreshold,
```

Comment:
We need also check if the param is legal or not.

Reply:
It checked in api.